modules/meilisearch/go.mod

```
@@ -0,0 +1,63 @@
+module github.com/testcontainers/testcontainers-go/modules/meilisearch
+
+go 1.23.1
```

blocker: all modules live in the lowest supported Go version, determined by https://endoflife.date/go
```suggestion
go 1.22
```